hrpc/call.go

```diff
@@ -43,6 +44,8 @@ type RegionClient interface {
 	Addr() string
 	QueueRPC(Call)
 	String() string
+	SetReadTimeout(time.Duration)
```

Comment:
there's probably no need for these functions in an interface, just pass the ReadTimeout as an argument to NewClient